build/Dockerfile

```diff
@@ -9,15 +9,18 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-FROM quay.io/bitnami/golang:1.14 as builder
-RUN go env GOPROXY
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/go-toolset
+FROM registry.access.redhat.com/ubi8/go-toolset:1.15.7-11 as builder
+ENV GOPATH=/go/
```

Comment:
do we need to mkdir /go so that golang can write here? 

Seeing this error in the PR check:

```
go: writing stat cache: mkdir /go: permission denied
```